common/thorhelper/thorsort.cpp

```diff
@@ -586,7 +588,7 @@ class TbbParallelMergeSorter
                 size_t start = posLeft[i] + posRight[i];
                 size_t end = posLeft[i+1] + posRight[i+1];
                 size_t num = end - start;
-                size_t numFwd = num/2;
+                size_t numFwd = (num+1)/2;
```

Comment:
trivial, but added 1 for consistency with other places.  It means it will merge 1 extra row forward rather than in reverse.